py/server/deephaven/jcompat.py

```diff
@@ -327,35 +326,6 @@ def _j_array_to_series(dtype: DType, j_array: jpy.JType, conv_null: bool) -> pd.
 
     return s
 
-def j_table_definition(table_definition: Union[Dict[str, DType], List[Column], None]) -> Optional[jpy.JType]:
```

Comment:
Was this internal before?

Reply:
It doesn't have a `_`, so it wasn't fully private / internal.  It is possible someone used it, so it would be a breaking change.

Reply:
I've added it back with deprecation for removal warning.